plot_test.go

```diff
@@ -56,10 +56,10 @@ func TestLegendAlignment(t *testing.T) {
 		},
 		&recorder.Fill{
 			Path: vg.Path{
-				{Type: vg.MoveComp, X: 80, Y: 30},
-				{Type: vg.LineComp, X: 80, Y: 40},
-				{Type: vg.LineComp, X: 100, Y: 40},
-				{Type: vg.LineComp, X: 100, Y: 30},
+				{Type: vg.MoveComp, X: 80, Y: 31.865107548701296},
```

Comment:
The constant is now 10.189054726368159 but this does not completely recover the previous behaviour.